server/src/lib/models/orders.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import mongoose from 'mongoose';
+import Customers from './customers';
+const Schema = mongoose.Schema;
+
+const orderSchema = Schema({
+  orderDate: { type: Date, required: true, default: Date() },
+  orderTotal: { type: Number },
+  itemQuantity: { type: Number, required: true, default: 1 },
+  shippingOption: { type: String, default: 'ground', enum: ['ground', '2-day air', 'freight'] },
+  customer: { type: Schema.Types.ObjectId, ref: 'Customers', required: true },
+});
+
+orderSchema.pre('findById', function (next) {
+  this.populate('customers');

```

Comment:
The string value passed to populate needs to match the name of the field on the schema you wish to populate, not the name of the collection. So, please update to 'customer'.